<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDataToFolderClass</class>
 <widget class="QDialog" name="AddDataToFolderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddDataToFolder</string>
  </property>
  <widget class="QTreeView" name="systemTree">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>591</width>
     <height>24</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="text">
     <string>Select files and folders to insert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="text">
     <string>Clear selected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fromExplorerButton">
    <property name="text">
     <string>Select from explorer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="fileList">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>1</y>
     <width>301</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
